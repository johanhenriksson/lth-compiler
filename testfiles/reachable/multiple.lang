int a() {
    b();
}

int b() {
    c();
}

int c() {
}
