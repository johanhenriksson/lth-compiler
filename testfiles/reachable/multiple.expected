digraph G {
  a;
  a -> b;
  a -> c;
  b;
  b -> c;
  c;
}
